Mathlib/MeasureTheory/Integral/BoundedContinuousFunction.lean

```diff
@@ -0,0 +1,134 @@
+/-
+Copyright (c) 2023 Kalle Kytölä. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kalle Kytölä
+-/
+import Mathlib.Analysis.InnerProductSpace.Basic
+import Mathlib.MeasureTheory.Integral.Bochner
+
+/-!
+# Integration of bounded continuous functions
+
+In this file, some results are collected about integrals of bounded continuous functions. They are
+mostly specializations of results in general integration theory, but they are used directly in this
+specialized form in some other files, in particular in those related to the topology of weak
+convergence of probability measures and finite measures.
+-/
+
+open MeasureTheory Filter
+open scoped ENNReal NNReal BoundedContinuousFunction Topology
+
+namespace BoundedContinuousFunction
+
+section NNRealValued
+
+variable {X : Type*} [MeasurableSpace X] [TopologicalSpace X] [OpensMeasurableSpace X]
+
+theorem NNReal.coe_ennreal_comp_measurable (f : X →ᵇ ℝ≥0) :
```

Comment:
You are in the `BoundedContinuousFunction` namespace, and you would like this kind of lemma to be accessible by dot notation, so making it long by prefixing it with `NNReal` is probably not a good pick. I would just call it `measurable_coe_ennreal_comp` as there is no ambiguity here.

Reply:
Thank you!

I thought that a sub-namespace would be clearer for organization, but I realize that it actually breaks the dot notation.

I wonder, however, if it can become a problem to state many lemmas that are specific to the case of `NNReal` codomain in the `BoundedContinuousFunction` namespace. I believe there are good use cases for those specific lemmas in the topology of weak convergence and in the Riesz-Markov-Kakutani representation theorem. But I mildly worry about littering the namespace with such particular cases...  I assume that the `..._of_...` is the best existing naming convention for specifying the codomain. So for now, essentially in line with your other suggestions, I will add something like `...of_nnreal...` to the names (and remove the attempted sub-namespace), except in cases which otherwise are unambiguous like this exact one.